### PR TITLE
fix(ci): fix `cmd/gossamer` unit tests

### PR DESCRIPTION
**Pull request by timwu20**
- Fixes the `cmd/gossamer` unit tests for the `v0.9.20` branch.
- The build spec command tests were using the wrong file.  Should be pointed to the raw build spec file.

## Tests

```sh
go test ./cmd/gossamer
```

## Issues

contributes to #2540 

## Primary Reviewer

@jimjbrettj

### CODE REVIEW

File: chain/dev/config.toml [hunk 1]
```diff
@@ -15,10 +15,10 @@ sync = ""
 digest = ""
 
 [init]
-genesis = "./chain/dev/dev-spec-raw.json"
+genesis = "./chain/dev/genesis-spec.json"
 
 [account]
-key = "alice"
+key = ""
```
Comment (qdm12) [inline chain/dev/config.toml:21]: Why is the key removed?

File: chain/gssmr/config-bob.toml [hunk 2]
```diff
@@ -1,49 +0,0 @@
-[global]
```
Comment (qdm12) [inline chain/gssmr/config-bob.toml:1]: Why is this file removed?

**Review by qdm12 — APPROVED** [review]

File: chain/gssmr/config-bob.toml (hunk 2 above)
Comment (timwu20) [reply chain/gssmr/config-bob.toml:1]: the config is expected to be general purpose.  You can specify accounts via command line.  This is just mirroring what it's like on `development`.

File: chain/dev/config.toml (hunk 1 above)
Comment (timwu20) [reply chain/dev/config.toml:21]: see [comment](https://github.com/ChainSafe/gossamer/pull/2560#discussion_r875106720)

**Review by EclesioMeloJunior — APPROVED** [review]
